bedrock/mozorg/redirects.py

```diff
@@ -222,6 +222,12 @@ def decider(request, **kwargs):
     redirect(r'^legal/eula/firefox-2/?$', '/about/legal/eula/firefox-2/'),
     redirect(r'^legal/eula/firefox-3/?$', '/about/legal/eula/firefox-3/'),
 
+    # Bug 1209643
+    redirect(r'^legal/bylaws_amendment_(?P<n>[12])(\.html|/)?', '/foundation/documents/bylaws-amendment-{n}/'),
```

Comment:
I did wonder if there was a way to do this as well, +1 from me in that it could be handy :+1: